### PR TITLE
feat: add service wild card search 

### PR DESCRIPTION
**Pull request by dustindemmerle**
Added Service Wild Card Search & DB tests for Service and IssueMatch

### CODE REVIEW

**Review by drochow — APPROVED** [review]

**Comment by drochow** [comment]
@dorneanu please take a look to for a second pair of eyes

**Review by michalkrzyz — DISMISSED** [review]

File: internal/database/mariadb/issue_match_test.go [hunk 1]
```diff
@@ -341,6 +341,75 @@ var _ = Describe("IssueMatch", Label("database", "IssueMatch"), func() {
 					}
 
 				})
+				It("can filter issue matches by PrimaryName using wild card search", func() {
```
Comment (michalkrzyz) [inline internal/database/mariadb/issue_match_test.go:344]: Is this copied from here: https://github.com/cloudoperators/heureka/blob/60dfd0af5d4c0abc4e5451024921b10de157e45a/internal/database/mariadb/issue_test.go#L329
It seems to be duplicated.

Comment (drochow) [reply internal/database/mariadb/issue_match_test.go:344]: @dustindemmerle ☝️

**Comment by drochow** [comment]
@dustindemmerle please sign your commits

**Review by drochow — APPROVED** [review]

**Pull request merged**